guides/gatherers.md

```diff
@@ -238,6 +239,44 @@ Example output (in Rhai):
 #{ nodeid: 2, ring0_addr: "10.80.1.12" };
 ```
 
+### fstab
+
+**Argument required**: no.
+
+This gatherer allows access to the /etc/fstab file, returning all entries available at the file.
```

Comment:
I know it is generic linux knowledge, but we could say that if  `backup` or `check_order` are not present in the fstab file, `0` will be returned